### ISSUE TITLE
Error:forkChoiceUpdateV1 called post-shanghai: BAD BLOCK: Block: 17034870

### ISSUE THREAD

**Issue by AlexanderGW** [ethereum/go-ethereum#27097]
#### System information

**Geth version:**
Version: 1.11.5-stable
Architecture: amd64
Go Version: go1.19.6

**CL client & version:**
Nimbus 23.3.2

**OS & Version:**
Linux, Fedora

**Commit hash :**
N/A

#### Expected behaviour
That the node continue syncing past block 17034869

#### Actual behaviour
Stuck at 17034869, calls 17034870 a "bad block"

````
WARN [04-12|23:42:43.721] Beacon client online, but no consensus updates received in a while. Please fix your beacon client to follow the chain!
WARN [04-12|23:39:54.988] Snapshot extension registration failed peer=2b289fb0 err="peer connected on snap without compatible eth support"
WARN [04-12|23:39:50.055] Checkpoint challenge timed out, dropping id=2be92ee3ae610bbe conn=dyndial addr=18.143.177.14:30303 type=Geth/v1.11.5-stable/...
WARN [04-12|23:37:37.774] NewPayloadV1: inserting block failed error="missing withdrawalsHash"

##############################

Receipts:

Chain config: &params.ChainConfig{ChainID:1, HomesteadBlock:1150000, DAOForkBlock:1920000, DAOForkSupport:true, EIP150Block:2463000, EIP150Hash:0x2086799aeebeae135c246c65021c82b4e15a2c451340993aacfd2751886514f0, EIP155Block:2675000, EIP158Block:2675000, ByzantiumBlock:4370000, ConstantinopleBlock:7280000, PetersburgBlock:7280000, IstanbulBlock:9069000, MuirGlacierBlock:9200000, BerlinBlock:12244000, LondonBlock:12965000, ArrowGlacierBlock:13773000, GrayGlacierBlock:15050000, MergeNetsplitBlock:<nil>, ShanghaiTime:(*uint64)(0xc0001b21c0), CancunTime:(*uint64)(nil), PragueTime:(*uint64)(nil), TerminalTotalDifficulty:58750000000000000000000, TerminalTotalDifficultyPassed:true, Ethash:(*params.EthashConfig)(0x2990f40), Clique:(*params.CliqueConfig)(nil)}

Platform: geth v1.11.5 go1.19.6 amd64 linux

Error: missing withdrawalsHash

Block: 17034870 (0x57f77052f67cdbdb14987b46d3a2d6f001dee0e974cb21da7f59c6dea4ad39c7)

########## BAD BLOCK #########

ERROR[04-12|23:37:37.774]

WARN [04-12|23:37:36.922] Ignoring already known beacon payload number=17,034,869 hash=c2558f..d85d8f age=10m13s
WARN [04-12|23:34:28.663] Beacon client online, but no consensus updates received in a while. Please fix your beacon client to follow the chain!
WARN [04-12|23:29:28.631] Previously seen beacon client is offline. Please ensure it is operational to follow the chain!
INFO [04-12|23:27:25.679] Stopping work on payload id=0x90a7eb95d8d8b04b reason=timeout
WARN [04-12|23:27:25.247] Served engine_forkchoiceUpdatedV1 reqid=365072 duration="115.127µs" err="Invalid parameters" errdata="{Error:forkChoiceUpdateV1 called post-shanghai}"
INFO [04-12|23:27:24.902] Chain head was updated number=17,034,869 hash=c2558f..d85d8f root=8b8da1..fdd295 elapsed=4.140891ms
````

#### Steps to reproduce the behaviour
Updated `geth` and `nimbus` 3 weeks ago, both had been running fine until the fork.

#### Notes
Related: https://github.com/ethereum/go-ethereum/issues/27091
With the exception of using Nimbus, instead of Lighthouse

Is there a way to pop this bad block off the stack, and have the node resync from `17034869`?

**Comment by gjgd** [comment]
I had the same issue using prysm running on Docker.

I was using this image which seems to be deprecated `gcr.io/prysmaticlabs/prysm/beacon-chain:stable`
Switching to this image [hosted on dockerhub ](https://hub.docker.com/layers/prysmaticlabs/prysm-beacon-chain/latest/images/sha256-522d82061ecaff1dcddec0bf1c85492e57d7fd979132739d5507a63fef12ee48?context=explore)fixed the sync issue `"prysmaticlabs/prysm-beacon-chain:latest"`

**Comment by MariusVanDerWijden** [comment]
Just updating your consensus layer node will fix this issue, nothing more needed, the bad block is not persisted (well some info is but we will accept the good version of it anyway)

**Comment by AlexanderGW** [comment]
@MariusVanDerWijden Updating what exactly? I'm already running the latest version of Nimbus CL client.

Are you saying their are manual steps that need to be carried out?

I've checked release notes for my CL client; there are no instructions of me having to do anything, aside from having the latest version installed.

I've tried restarting Geth and the Nimbus CL client, even in different orders, neither are able to sync.

The logs of both clients are reporting low peer counts, neither are able to proceed as something has clearly gone wrong somewhere.
